docs/source/how-to/custom-formats.md

```diff
@@ -142,13 +285,14 @@ First consider the section
 
 ```yaml
 media_types:
-  dataframe:
-    application/x-xdi: exporter:serialize_xdi
+  xdi:
```

Comment:
We had talked about separate namespaces here, as in

```yaml
media_types:
    by_structure_family:
        ...
    by_spec:
        xdi:
```

Do we still want to go that way?

Reply:
Follow-up thought: this may require a fairly significant refactor. I'd be fine with using a single namespace for now and separating them in a future PR.

Reply:
makes sense to me. lets leave it for now